source/manual/alerts/search-spelling-suggestions.html.md

```diff
@@ -0,0 +1,23 @@
+---
+owner_slack: '#search-team'
+review_by: 2017-06-26
+title: 'Test spelling suggestions work for example queries failed'
```

Comment:
Yeah, this error message is a bit clunky because Jenkins adds the word "failed" to the service name. I've used your suggestion and updated alphagov/govuk-puppet#5836.